try {
  console.log(foo);
} 
catch {
  console.log("Terjadi error...");
} 
finally {
  console.log("Ini akan selalu di jalankan");
}
